src/main/java/org/apache/commons/collections4/CollectionUtils.java

```diff
@@ -624,9 +624,7 @@ public static boolean isEqualCollection(final Collection<?> a, final Collection<
      */
     public static <E> int hashCode(final Collection<? extends E> collection,
                                 final Equator<? super E> equator) {
-        if (null == equator) {
-            throw new NullPointerException( "Equator must not be null." );
-        }
+        Objects.requireNonNull(equator, "Equator");
```

Comment:
"Equator" -> "equator"

Reply:
Thanks @garydgregory . I had updated Equator to equator .